README.md

```diff
@@ -3,7 +3,7 @@
 [![Rust](https://github.com/Kuadrant/wasm-shim/actions/workflows/rust.yaml/badge.svg)](https://github.com/Kuadrant/wasm-shim/actions/workflows/rust.yaml)
 [![FOSSA Status](https://app.fossa.com/api/projects/custom%2B162%2Fgit%2Bgithub.com%2FKuadrant%2Fwasm-shim.svg?type=shield&issueType=license)](https://app.fossa.com/projects/custom%2B162%2Fgit%2Bgithub.com%2FKuadrant%2Fwasm-shim?ref=badge_shield&issueType=license)
 
-A Proxy-Wasm module written in Rust, acting as a shim between Envoy and Limitador.
+A Proxy-Wasm module written in Rust, acting as a shim between Envoy and Limitador & Authorino.
```

Comment:
I guess there's also the option to not mention the services themselves as it isn't necessarily specific to limitador and authorino:
```suggestion
A Proxy-Wasm module written in Rust, acting as a shim between Envoy and both Rate-limiting and External Authorization services.
```

Reply:
LGTM